octobot_evaluators/api/evaluators.py

```diff
@@ -195,6 +196,10 @@ async def create_all_type_evaluators(tentacles_setup_config: object,
                                      real_time_time_frames: list = None,
                                      relevant_evaluators=CONFIG_WILDCARD,
                                      ) -> list:
+    if not get_activated_strategies_classes(tentacles_setup_config):
+        # If no strategy is activated, there is no evaluator to create (their evaluation would not be used)
+        get_logger(LOGGER_NAME).info(f"No evaluator to create for {exchange_name}: no activated evaluator strategy.")
```

Comment:
I am wondering about this log level. It's an uncommon situation, it's maybe better to warn the user with a warning level. What do you think?

Reply:
If there is a warning to print I think we should print it in the trading mode, when a required strategy is not activated. Here It's a normal behaviors on staggered orders, tradingview and arbitrage trading modes

Reply:
OK 👍 I just cant figure out why staggered and trading view are concerned.

Reply:
They don't use evaluators either (evaluator might still be activated in user config when using these trading modes) therefore it's better not to create them at all

Reply:
Oh yes my bad.